lib/prism.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const Prism = require('prismjs');
+const prismLoadLanguages = require('prismjs/components/');
+
+// https://github.com/PrismJS/prism/issues/2145
+const prismComponents = require('prismjs/components');
+const prismSupportedLanguages = Object.keys(prismComponents.languages);
+
+const escapeHTML = require('./escape_html');
+const unescapeHTML = require('./unescape_html');
+
+/**
+ * Wrapper of Prism.highlight()
+ * @param {String} code
+ * @param {String} language
+ */
+function prismHighlight(code, language) {
+  console.log(Object.keys(Prism.languages));
+  // Prism has not load the language pattern
+  if (!Prism.languages[language] && prismSupportedLanguages.includes(language)) prismLoadLanguages(language);
+
+  if (Prism.languages[language]) {
+    // Prism escapes output by default
+    return Prism.highlight(unescapeHTML(code), Prism.languages[language], language);
```

Comment:
If `escapeHTML(code)` in the later line is necessary, `code` will be a plain text (no escaped).
So I thnk this `unescapeHTML()` will not be necessary.

Reply:
We cannot identified whether the `code` is escaped or not, so unescape the code before highlighting, and always return the escaped result.

Reply:
`unescapeHTML()` breaks a sample HTML code such as `"&" => &amp;`.
(The result will look like `"&" => &`.  This will not be intended by the author.)
I think we should believe that `code` is a plain text.

Reply:
`unescapeHTML` is removed and rebased.